frontend/src/pages/NewExperiment.test.tsx

```diff
@@ -53,37 +54,35 @@ describe('NewExperiment', () => {
     createExperimentSpy.mockImplementation(() => ({ id: 'new-experiment-id' }));
   });
 
-  it('renders the new experiment page', () => {
-    const tree = shallow(<NewExperiment {...generateProps() as any} />);
+  afterEach(() => tree.unmount());
 
+  it('renders the new experiment page', () => {
+    tree = shallow(<NewExperiment {...generateProps() as any} />);
```

Comment:
_**[suggestion]** Since all methods depend on a shared global `tree` there is no reason to replicate the construction per test case. Maybe make a lambda producer `newTree()`_ like so:
```js
it('renders the new experiment page', () => {
    newTree();
```
while `newTree()` is implemented as:
```js
const newTree = _ => shallow(<NewExperiment {...generateProps() as any} />)
```

Reply:
Even though this doesn't seem to be the case for this file, the way the component is mounted is a decision made in each test individually, and they can be very different (deep vs shallow mounts, changing props... etc). I'd rather keep this consistent with other tests where mounting is done explicitly in each test.